### PR TITLE
Migration to drone

### PR DESCRIPTION
**Pull request by MorrisJobke**
first version of drone.yml

### CODE REVIEW

**Comment by AndyScherzinger** [comment]
LGTM :+1:

**Comment by AndyScherzinger** [comment]
@MorrisJobke the travis build keep failing since travis just tries to build them and defaults to ruby... :(

**Comment by MorrisJobke** [comment]
I rebased to check if the tests now pass ;)

**Comment by AndyScherzinger** [comment]
:+1:

File: build.gradle [hunk 1]
```diff
@@ -10,7 +10,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.3'
+        classpath 'com.android.tools.build:gradle:2.2.0'
```
Comment (MorrisJobke) [inline build.gradle]: Best would be to update the docker container too ;)

Comment (MorrisJobke) [inline build.gradle]: See https://github.com/nextcloud/docker-ci/pull/17

**Pull request merged**